src/app/components/PageHeader/PageHeader.tsx

```diff
@@ -43,42 +69,16 @@ const ContentHeader: FunctionComponent<ContentHeaderProps> = ({
 				align-items: center;
 				background-color: ${theme.colors.white};
 			`}
+			className={className}
```

Comment:
Not necessary for this PR, as nobody is "reusing" this: I've gotten a habit of using the `classnames` package for reusable components, injecting a `csx('PageHeader', className)` pattern, so it has a default local name, aside from the ones passed in (useful when someone wants to target an element through CSS)